### DISCUSSION TITLE
Pot | 常规设置

### DISCUSSION THREAD

**Discussion by giscus** [pot-app/pot-desktop#627] [General]
# Pot | 常规设置

跨平台划词翻译和OCR软件

https://pot-app.com/docs/config/general.html

**Comment by lhtuling** [comment]
代理并不能正常使用，而且代理不能输入账号密码

**Reply by wanglei52076** [reply]
可以使用的 带用户名密码的直接：http://user:password@ip/域名 然后单独端口就行了！如果本地有代理软件一般都支持http代理 看一下代理端口 地址http://127.0.0.1 前提是客户端没有将这个代理排除

**Comment by OblGu** [comment]
开启了“监听剪贴板”之后，如何避免在复制翻译结果的时候，软件再次读取复制内容并翻译一遍

**Comment by elarbor** [comment]
真希能加入自动升级的功能，不然还得在github里下载😂
